wing.py

```diff
@@ -344,11 +344,8 @@ def setup(self, **kwargs):
                 taper >= 0.15, # TODO
 
                 # Fuel volume [TASOPT doc]
-##                TCS([Afuel <= wwn*0.92*tau]),
-                # GP approx of the signomial constraint:
-
-                #.3026 computed from TASOPT 737 case, assumes all lengths in m
-                Vfuel <= 0.3026*mac**2*b*tau,
+                TCS([Afuel <= wwn*0.92*tau]),
+                Vfuel <= croot**2*b/6*(1+taper+taper**2)*cosL, # [SP]
                 WfuelWing <= rhofuel*Vfuel*g,
```

Comment:
@mayork You're right about line 349. Does it still cause problems if that is fixed? I have a hard time understanding why this model breaks things so badly. Let's discuss more tomorrow.